app/javascript/components/form/fields/checkbox-input.jsx

```diff
@@ -23,7 +23,9 @@ const CheckboxInput = ({ commonInputProps, options, metaInputProps }) => {
       error={error}
       className={css.checkboxContainer}
     >
-      <FormLabel required={required}>{label}</FormLabel>
+      <FormLabel component="legend" required={required}>
```

Comment:
Does this change how the label looks? If not, this is ok

Reply:
I don't think so. But  I will verify it. Thanks

Reply:
No, it doesn't change the look

Reply:
ok cool, I'll merge first thing tomorrow